overlay/real-host-triple.nix

```diff
@@ -5,7 +5,7 @@ let
   kernel = platform.parsed.kernel.name;
   abi = platform.parsed.abi.name;
 in
-if platform.isWasi then
+if platform.isWasi or false then
```

Comment:
Nice catch!
As a note, the `isWasi` symbol is available in upstream `nixpkgs` since June.

Reply:
This is a huge help! I've been having issues building `cargo2nix` projects in NixOS environments using very old Nixpkgs (e.g. `nix-docker`) specifically because of this attribute not being present.